include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -474,6 +474,9 @@ layout = [0  4  8  12]
          [3  7  11 15]
 
 For the Threads Per Warp and Values Per Thread level, the linear id distribution is variant for each sub-class encoding.
+
+If the layout does not completely cover the tensor, we tile it until we cover the entire tensor.
+The order of this tiling is, incidentally, the same as the order of the elements, although this may change in the future.
```

Comment:
Is this true for AMD layouts @zhanglx13? If not, what can we say about the tiling order of distributed layouts in general?

Reply:
Unfortunately, this is not the case for AMDMfma layout when isTransposed=False.
How about we say "the order of this tiling is the same as the order of how warps are distributed in each CTA"?

Reply:
Sadly, that wouldn't be correct for `wgmma`, for which the warps are in M-major order (due to the warp-group requirement) and then it's packed on an N-major order
https://github.com/triton-lang/triton/blob/1070ca263b6062ef2de8595da1db01fce98b07ff/third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/DotOpToLLVM/WGMMA.cpp#L423-L424

I think that the best we can do here is to just add that AMDMfma(isTransposed=False) is an exception
A better long-term solution would be to add a method `getTilingOrder` that returns this property.

Reply:
Actually, I implemented this in https://github.com/triton-lang/triton/pull/5089/commits/522bce05e0237a00e2a575e0b6c634b51085ea0a for all but AMD layouts. I'll let the AMD guys implement it for theirs if they deem this useful.

Reply:
Agreed that `getRepOrder` is much more robust. We'll work on the implementation for AMD layout in a future PR. Thanks @lezcano